src/lib.rs

```diff
@@ -124,6 +124,31 @@ pub struct Wizer {
     #[cfg_attr(feature = "structopt", structopt(long = "allow-wasi"))]
     allow_wasi: bool,
 
+    /// Provide an additional preloaded module that is available to the
+    /// main module.
+    ///
+    /// This allows running a module that depends on imports from
+    /// another module. Note that the additional module's state is *not*
+    /// snapshotted, nor is its code included in the Wasm snapshot;
+    /// rather, it is assumed that the resulting snapshot Wasm will also
+    /// be executed with the same imports available.
+    ///
+    /// The main purpose of this option is to allow "stubs" for certain
+    /// intrinsics to be included, when these will be provided with
+    /// different implementations when running or further processing the
+    /// snapshot.
+    ///
+    /// The format of this option is `name=file.{wasm,wat}`; e.g.,
+    /// `intrinsics=stubs.wat`.
+    #[cfg_attr(feature = "structopt", structopt(long = "preload"))]
+    preload: Option<String>,
+
+    /// Like `preload` above, but with the module contents provided,
+    /// rather than a filename. This is more useful for programmatic
+    /// use-cases where the embedding tool may also embed a Wasm module.
+    #[cfg_attr(feature = "structopt", structopt(skip))]
+    preload_bytes: Option<(String, Vec<u8>)>,
```

Comment:
I think both of these should be `Vec`s rather than `Option`s so that rather than 0 or 1 preloads we can have any number of preloads.

Reply:
Done!